src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -406,6 +406,9 @@ function ComposerWithSuggestions(
      */
     const updateComment = useCallback(
         (commentValue: string, shouldDebounceSaveComment?: boolean) => {
+            if (isIOSNative) {
+                syncSelectionWithOnChangeTextRef.current = null;
+            }
```

Comment:
https://github.com/Expensify/App/blob/2514f2988eb26858d30fa501c0040b1f8fc2e239/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx#L442-L444

I think your main solution to add additional here should work, right?
```js
if (commentWithSpaceInserted !== newComment && isIOSNative) {
```

Reply:
My main solution is actually to use it here.
https://github.com/Expensify/App/blob/24c5fe25867e9a6fd2b5232060f0d548f765678f/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx#L438

That works too though. Which one do you prefer?

Reply:
I think adding additional check here is better 

https://github.com/Expensify/App/blob/24c5fe25867e9a6fd2b5232060f0d548f765678f/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx#L438

Reply:
Updated!